static/templates/settings/display_settings.hbs

```diff
@@ -59,6 +59,12 @@
                             <img class="emoji" src="/static/generated/emoji/images-{{this.key}}-64/1f389.png" />
                             {{/if}}
                         </span>
+                        {{#if (eq this.key "google-blob")}}
+                        <span><em>(deprecated)</em></span>
```

Comment:
"deprecated", like any other user facing text, needs to be tagged for translation. I also moved the `(` outside the `em` tag, to match how our other settings with this pattern do it.